common-test-lib/src/main/java/com/google/gct/idea/testing/TestUtils.java

```diff
@@ -127,6 +129,7 @@ public static MockProject mockProject(@Nullable PicoContainer container) {
     return new MockProject(container, getParentDisposableForCleanup());
   }
 
+  @SuppressFBWarnings(value="DM_STRING_CTOR", justification="Warning due to string concatenation")
```

Comment:
Also, there are other warnings in a similar line, e.g., complaining checking for not-null unnecessarily whenever a reference is returned from a method, so being able to look into bytecode somehow would be useful for determining the root cause of other similar warnings.